cashu/mint/main.py

```diff
@@ -4,7 +4,7 @@
 import uvicorn
 from click import Context
 
-from ..core.settings import settings
+from ..core.settings import settings, VERSION
```

Comment:
I think this should fix the lint error in the CI  (unsorted imports)

```suggestion
from ..core.settings import VERSION, settings
```